plugins/essif-lab_contactform7/admin/class-essif-lab_contactform7-admin.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * The admin-specific functionality of the plugin.
+ *
+ * @link       #
+ * @since      1.0.0
+ *
+ * @package    Essif_Lab_contactform7
+ * @subpackage Essif_Lab_contactform7/admin
+ */
+
+/**
+ * The admin-specific functionality of the plugin.
+ *
+ * Defines the plugin name, version, enqueue the admin-specific stylesheet and JavaScript.
+ *
+ * @package    Essif_Lab_contactform7
+ * @subpackage Essif_Lab_contactform7/admin
+ * @author     Duur Klop, Luuk van Houdt, Ruben Sikkens, Ufuk Altinçöp en Weis Mateen <ruben.sikkens@student.hu.nl>
+ */
+class Essif_Lab_contactform7_Admin {
```

Comment:
Zou hier eerder Admin van maken en gebruik maken van namespaces zie ook hoe het gedaan is in de hoofdplugin.